service/src/main/java/uk/gov/hmcts/reform/fpl/service/TaskListRenderer.java

```diff
@@ -117,20 +122,27 @@ public String render(List<Task> allTasks) {
 
         section.add(renderHorizontalLine());
         sec.getTasks().forEach(task -> {
-            section.addAll(renderTask(task));
+            section.addAll(renderTask(task, showInProgressTag));
             section.add(renderHorizontalLine());
         });
 
         return section;
     }
 
-    private List<String> renderTask(Task task) {
+    private List<String> renderTask(Task task, boolean showInProgress) {
         final List<String> lines = new LinkedList<>();
 
         switch (task.getState()) {
```

Comment:
what if toggle change in the middle? Not quite sure how it affect performance, definitely it can affect LD statistics which shows how many times toggle has been evaluated. This code basically sits inside loop